tests/ha/test_multi_server_pod_failure.py

```diff
@@ -2810,3 +2810,317 @@ def test_reads_writes_during_kserver_pods_down(self):
+                                                               hostname=hostname)
+            assert_utils.assert_true(resp[0], resp[1])
+            server_data_pod_list.remove(pod_name)
+            LOGGER.info("Step 4: Services of deleted server pod %s are in offline state", pod_name)
+
+            LOGGER.info("Step 5: Check services status on remaining pods %s", server_data_pod_list)
+            resp = self.hlth_master_list[0].get_pod_svc_status(pod_list=server_data_pod_list,
+                                                               fail=False)
+            LOGGER.debug("Response: %s", resp)
+            assert_utils.assert_true(resp[0], resp)
+            LOGGER.info("Step 5: Services of remaining pods are in online state")
+
+            LOGGER.info("Step 6: Perform IOs with %s server pods down", count)
+            resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                        log_prefix=self.test_prefix,
+                                                        skipcleanup=True, setup_s3bench=False,
+                                                        nsamples=1, nclients=1)
+            assert_utils.assert_true(resp[0], resp[1])
+            LOGGER.info("Step 6: IOs run successfully with %s server pods down", count)
+
+        LOGGER.info("Step 7: Create new user and run IOs with %s server pods down", value)
+        users = self.mgnt_ops.create_account_users(nusers=1)
+        self.test_prefix = 'test-42054-1'
+        self.s3_clean.update(users)
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=self.test_prefix,
+                                                    nsamples=1, nclients=1, setup_s3bench=False)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 7: IOs run successfully with new user with single server pod running.")
+
+        LOGGER.info("Completed: Test to verify IOs after each pod failure till only "
+                    "single server pod is running.")
+
+    # pylint: disable=C0321
+    # pylint: disable=too-many-locals
+    # pylint: disable=unused-argument
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-42053")
+    @CTFailOn(error_handler)
+    def test_conti_ios_till_single_server_pod_running(self):
+        """
+        This test tests continuous READs/WRITEs/DELETEs while server pods are failing till only one
+        server pod is running.
+        """
+        LOGGER.info("STARTED: Test continuous READs/WRITEs/DELETEs while server pods are failing "
+                    "till only one server pod is running")
+
+        event = threading.Event()  # Event to be used to send when server pods going down
+        wr_bucket = self.kvalue * 5 + HA_CFG["s3_bucket_data"]["no_buckets_for_deg_deletes"]
```

Comment:
This should be value not kvalue right?

Reply:
This is just to make extra buckets, so we not run outof bucket for delete. I guess this will just add few more bucket. Won't cause any issue.

Reply:
I think even if we just have kvalue * 5 , it will be sufficient for number buckets to be deleted.

Reply:
Then do we really need to multiply any value with 5 and add 150?
Only 150 won't be sufficient?